src/ngraph/runtime/cpu/pass/cpu_assignment.cpp

```diff
@@ -831,7 +831,8 @@ static const runtime::cpu::pass::AssignOpMap s_dispatcher{
      &runtime::cpu::pass::CPUAssignment::assign<ngraph::op::QuantizedAvgPool>},
     {TI(ngraph::op::Softmax), &runtime::cpu::pass::CPUAssignment::assign<ngraph::op::Softmax>},
     {TI(ngraph::op::Slice), &runtime::cpu::pass::CPUAssignment::assign<ngraph::op::Slice>},
-    {TI(ngraph::op::Quantize), &runtime::cpu::pass::CPUAssignment::assign<ngraph::op::Quantize>},
+    {TI(ngraph::op::QuantizeCPU),
```

Comment:
`CPUQuantize` ? I think the prefix convention is more common. I defer to @jbobba

Reply:
I think the idea is that we would delete this operator and @nbpatel would rework his kernels to use the nGraph (vs. CPU) Quantize op.  So, given this is temporary maybe it's not such a big deal.  However, I can rename to what @jbobba and @nbpatel decide.